src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -366,6 +366,12 @@ private static void validateCheckSection(ModuleFactory moduleFactory, String fil
             final String subSectionName = subSection.getAttributes().getNamedItem("name")
                     .getNodeValue();
 
+            // following subsections of Checker are not typical
+            if ("Checker".equals(sectionName)
+                    && ("Example of Usage".equals(subSectionName)
+                        || "Metadata".equals(subSectionName))) {
```

Comment:
Shouldn't we move this line to comment `can be in different orders, and completely optional` which is right below this?